src/Sarif.Viewer.VisualStudio/CodeLocationObject.cs

```diff
@@ -76,20 +68,20 @@ public virtual string FilePath
         {
             get
             {
-                return _filePath;
+                return this._filePath;
             }
             set
             {
-                if (value != _filePath)
+                if (value != this._filePath)
```

Comment:
Is it not better to use !String::Equals as the instances can be different but the content the same?